server/models/handlers/account.js

```diff
@@ -1,9 +1,24 @@
 /* The goal: Take in the user input from the Account Creation form in 
 the front end UI in order to create a user account in the backend*/
 
-async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
-    
-    const user = {
+async function createUserAccount(first_name, last_name, type, student_id, email, password, phone) {
+     // Searches the student table to see if a student account exists for the given data.
+
+     If userType = userType.student {
+     const user = await studentModel.insertStudentAccount(first_name, last_name, type, student_id, email, password, phone);
```

Comment:
Hi Nancy, 
I would recommend right clicking and using the Format option in Visual Studio Code to align the code to increase code readability.

Reply:
Done :)